internal/core/subsume/testdata/subsume.txtar

```diff
@@ -0,0 +1,4837 @@
+-- out/TestValues/0/__⊑__/v2 --
```

Comment:
I don't see any test inputs here. Am I missing something?

Reply:
This is driven by the existing test cases in https://github.com/cue-lang/cue/pull/3862/files#diff-5c570d819733910b5664134701d7cb95c9efbf2261b72b32b3ff5339ef18bb27L52-R77.
What I wanted to do was re-use all of those test cases and then use the txtar tooling for rendering the error messages.

Reply:
We typically want to keep the tests together. So either put input an output in txtar, or do everything in a table.

Also, we prefer not to change the general design of test. It is a lot of code to review! Finally, we want to separate out semantic from structural changes. So if we want to change the generic test structure, this should be a separate CL.